bot/commands/misc/convert.js

```diff
@@ -0,0 +1,52 @@
+// Converts from one currency to another based on this API: http://exchangeratesapi.io/
+
+const {Command} = require('yuuko');
+const fetch = require('node-fetch');
+const convert = require('convert-units');
+
+function convertUnits (msg, baseValue, baseType, targetType) {
+	const conversion = convert(baseValue).from(baseType).to(targetType);
+	msg.channel.createMessage(`${baseValue}${baseType} is equal to ${conversion}${targetType}`);
+}
+
+async function convertCurrency (msg, baseValue, baseType, targetType, prefix) {
```

Comment:
Same as above for this function too - let it just return a string or throw an error, and do the messaging in the command handler. You can eliminate the `msg` and `prefix` arguments here.